GCRCatalogs/instance_catalog.py

```diff
@@ -266,7 +266,7 @@ def _generate_native_quantity_list(self):
         return native_quantities
 
     def _pd_read_table(self, obj_type, **kwargs):
-        return pd.read_table(
+        return pd.read_csv(
```

Comment:
From the documentation, it looks like read_csv and read_table behave identically when delim_whitespace is True.   Why did you change it?

Reply:
`read_table` was deprecated at one point but apparently it has been "un-deprecated" now (see https://github.com/pandas-dev/pandas/issues/25220). 

The version of pandas that is currently in our `desc-stack` is not up-to-date enough so it still prints out a deprecation warning for `read_table`. However, it's true that this edit is not necessary.

Reply:
Thanks; that explains it.